.evergreen/start-orchestration.sh

```diff
@@ -14,37 +14,17 @@ MONGO_ORCHESTRATION_HOME="$1"
 echo From shell `date` > $MONGO_ORCHESTRATION_HOME/server.log
 
 cd "$MONGO_ORCHESTRATION_HOME"
-# Setup or use the existing virtualenv for mongo-orchestration.
-# Prefer using Python 3 from the toolchain over the default system python.
-# We may still fallback to using an old version of Python here.
-# Many of the Linux distros in Evergreen ship Python 2.6 as the
-# system Python. Core libraries installed by virtualenv (setuptools,
-# pip, wheel) have dropped, or soon will drop, support for Python
-# 2.6. Starting with version 14, virtualenv upgrades these libraries
-# to the latest available on pypi when creating the virtual environment
-# unless you pass --never-download.
-PYTHON=$(command -v /opt/mongodbtoolchain/v2/bin/python3 || command -v python3 || command -v python)
-$PYTHON -c 'import sys; print(sys.version)'
-$PYTHON -c 'import ssl; print(ssl.OPENSSL_VERSION)'
-
-if [ -f venv/bin/activate ]; then
-  . venv/bin/activate
-elif [ -f venv/Scripts/activate ]; then
-  . venv/Scripts/activate
-elif $PYTHON -m virtualenv --system-site-packages --never-download venv || $PYTHON -m venv --system-site-packages venv || virtualenv --system-site-packages --never-download venv; then
-  if [ -f venv/bin/activate ]; then
-    . venv/bin/activate
-  elif [ -f venv/Scripts/activate ]; then
-    . venv/Scripts/activate
-  else
-    echo "Unable to create virtual environment"
-    exit 1
-  fi
-fi
+
+. "$DRIVERS_TOOLS/.evergreen/find-python3.sh"
+. "$DRIVERS_TOOLS/.evergreen/venv-utils.sh"
```

Comment:
Any way to avoid adding a dependency on DRIVERS_TOOLS? Adding this makes it harder to run this script locally.

Reply:
I assumed `$DRIVERS_TOOLS` is defined at this point because `start-orchestration.sh` is invoked by `run-orchestration.sh`, which [already assumes](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/10f1a128f09a80e4d4ac42579a0f17b81338ab81/.evergreen/run-orchestration.sh#L102) it is defined. So I do not think this is necessary "adding" a dependency, unless there exist use cases where `start-orchestration.sh` is invoked directly instead of via `run-orchestration.sh`.

Reply:
In that case this is fine.

Reply:
Was there any reason this didn't refer to `find-python3.sh` by a relative path, as was done in various other `*venv.sh` scripts (e.g. [.evergreen/ocsp/activate-ocspvenv.sh in a3a17f1]( https://github.com/mongodb-labs/drivers-evergreen-tools/commit/a3a17f1185643135a6b24f318bf0ce1e58fdea94#diff-f280ef7a2bca6cfda5c70649f52d1e7013583a4551ee1348c371d6498921d1f6R29))?

Reply:
The `venv` scripts already required being invoked from within the same directory as the script file. `start-orchestration.sh` (or rather, `run-orchestration.sh`) did not appear to have such a requirement, so I did not want to introduce such a requirement when users may be invoking the script from different working directories.

However, the Bash shell requirement should allow us to use `BASH_SOURCE` as a more reliable alternative to relative paths. Created https://github.com/mongodb-labs/drivers-evergreen-tools/pull/257 in response.